src/plotSearch/components/plotSearchSections/basicInfo/PlotSearchSiteEditCustomDetailedPlan.js

```diff
@@ -0,0 +1,280 @@
+            </FormTextTitle>
+            <FormText>
+              {get(currentCustomDetailedPlan, 'preconstruction_estimated_construction_readiness_moment') || '-'}
+            </FormText>
+          </Column>
+          {infoLinks.length > 0 && (<Column small={12} medium={12} large={12} className="plot_search_target__info-links">
+            <Row>
+              <Column small={12} medium={4} large={4}>
+                <FormTextTitle>
+                  {PlotSearchFieldTitles.INFO_LINK_DESCRIPTION}
+                </FormTextTitle>
+              </Column>
+              <Column small={8} medium={6} large={6}>
+                <FormTextTitle>
+                  {PlotSearchFieldTitles.INFO_LINK_URL}
+                </FormTextTitle>
+              </Column>
+              <Column small={4} medium={2} large={2}>
+                <FormTextTitle>
+                  {PlotSearchFieldTitles.INFO_LINK_LANGUAGE}
+                </FormTextTitle>
+              </Column>
+            </Row>
+            {infoLinks.map((infoLink) => <Row key={infoLink.id}>
+              <Column small={12} medium={4} large={4}>
+                <FormText>
+                  {infoLink.description}
+                </FormText>
+              </Column>
+              <Column small={8} medium={6} large={6}>
+                <FormText>
+                  <ExternalLink href={infoLink.url} text={infoLink.url} />
+                </FormText>
+              </Column>
+              <Column small={4} medium={2} large={2}>
+                <FormText>
+                  {getInfoLinkLanguageDisplayText(infoLink.language, attributes)}
+                </FormText>
+              </Column>
+            </Row>)}
+          </Column>)}
+          {usageDistributions.length > 0 && (<Column small={12} medium={12} large={12} className="plot_search_target__info-links">
+            <Row>
+              <Column small={12} medium={4} large={4}>
+                <FormTextTitle>
+                  {PlotSearchFieldTitles.USAGE_DISTRIBUTION}
+                </FormTextTitle>
+              </Column>
+              <Column small={8} medium={6} large={6}>
+                <FormTextTitle>
+                  {PlotSearchFieldTitles.USAGE_DISTRIBUTION_BUILD_PERMISSION}
+                </FormTextTitle>
+              </Column>
+              <Column small={4} medium={2} large={2}>
+                <FormTextTitle>
+                  {PlotSearchFieldTitles.USAGE_DISTRIBUTION_NOTE}
+                </FormTextTitle>
+              </Column>
+            </Row>
+            {usageDistributions.map((usageDistribution) => <Row key={usageDistribution.distribution}>
+              <Column small={12} medium={4} large={4}>
+                <FormText>
+                  {usageDistribution.distribution}
+                </FormText>
+              </Column>
+              <Column small={8} medium={6} large={6}>
+                <FormText>
+                  {usageDistribution.build_permission}
```

Comment:
This is also missing the unit.

Reply:
This one is better if we can assume this is always set. I didn't notice before that the usage distribution fields lack the fallbacks, but if they should always be filled, then they're probably not needed.